crates/router/src/connector/cybersource/transformers.rs

```diff
@@ -261,7 +261,34 @@ pub struct CybersourceConsumerAuthInformation {
     xid: Option<String>,
     directory_server_transaction_id: Option<Secret<String>>,
     specification_version: Option<String>,
+    /// This field specifies the 3ds version
+    pa_specification_version: Option<PaSpecificationVersion>,
+    /// Verification response enrollment status.
+    ///
+    /// This field is supported only on Asia, Middle East, and Africa Gateway.
```

Comment:
What about the US gateway? Should we skip sending this value or external authentication itself is not available?